### PR TITLE
Update documentation for `Running tests`

### PR DESCRIPTION
**Pull request by jugmac00**
- use Python 3.9 instead of unsupported Python 2.7
- update `tox` command to enable running a single test

Previously, all tests were selected and coverage ran, which makes no
sense if you only run one test.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -19,7 +19,7 @@ deps =
 install_command = pip install --pre {opts} {packages}
 commands =
     python --version
-    py.test {posargs}
+    pytest {posargs}
```
Comment (jugmac00) [inline tox.ini:22]: This change is unrelated. Though, pytest dropped the dot a couple of years ago.

File: tox.ini [hunk 2]
```diff
@@ -5,7 +5,7 @@ skip_missing_interpreters = True
 [testenv]
 commands =
     python --version
-    pytest tests --cov-report term-missing --cov-fail-under 100 --cov cornice {posargs}
+    pytest {posargs: --cov-report term-missing --cov-fail-under 100 --cov cornice}
```
Comment (jugmac00) [inline tox.ini:8]: Everything behind `posargs:` is the default value which can be overridden - this is necessary to be able to select a single test.

File: docs/source/testing.rst [hunk 3]
```diff
@@ -11,11 +11,7 @@ just setup ``tox`` and run it inside the toplevel project directory::
 
 To run a single test inside a specific Python environment, do e.g.::
 
-    tox -e py27 tests/test_validation.py:TestServiceDefinition.test_content_type_missing
-
-or::
-
-    tox -e py27 tests.test_validation:TestServiceDefinition.test_content_type_missing
+    tox -e py39 tests/test_validation.py::TestServiceDefinition::test_content_type_missing
```
Comment (jugmac00) [inline docs/source/testing.rst:14]: I was only able to make this variant to run. Also most people possibly prefer to use `-k` to filter the tests.

File: tox.ini (hunk 2 above)
Comment (leplatrem) [reply tox.ini:8]: oH I had no idea of this, thanks!
